index.html

```diff
@@ -3,20 +3,54 @@
   <head>
     <meta charset="utf-8">
     <title>ACLU Cash Bail</title>
+    <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/semantic-ui@2.4.2/dist/components/transition.min.css">
+    <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/semantic-ui@2.4.2/dist/components/dropdown.min.css">
     <link rel="stylesheet" href="src/css/style.css">
     <link rel="stylesheet" href="src/css/table.css">
+    <link rel="stylesheet" href="src/css/search.css">
   </head>
   <body>
     <div class="container">
-      <table id="bail-rate-table">
-        <thead></thead>
-        <tbody></tbody>
-      </table>
-      <table id="bail-race-table">
-        <thead></thead>
-        <tbody></tbody>
-      </table>
-    </div>
-    <script src="src/js/index.js" charset="utf-8"></script>
+      <div id="bail-rate-container">
+        <div class="ui fluid multiple search selection dropdown">
+          <input type="hidden" name="county">
+          <i class="dropdown icon"></i>
+          <div class="default text">Select County</div>
+          <div class="menu"></div>
+        </div>
+        <table id="bail-rate-table">
+          <thead></thead>
+          <tbody></tbody>
+        </table>
+      </div>
+      <div id="bail-race-container">
+        <div class="ui fluid multiple search selection dropdown">
+          <input type="hidden" name="county">
+          <i class="dropdown icon"></i>
+          <div class="default text">Select County</div>
+          <div class="menu"></div>
+        </div>
+        <table id="bail-race-table">
+          <thead></thead>
+          <tbody></tbody>
+        </table>
+      </div>
   </body>
+    <script src="https://cdn.jsdelivr.net/npm/jquery@3.5.1/dist/jquery.slim.js"></script>
```

Comment:
Hmm, do we have to use jQuery? I have a personal vendetta...

Reply:
Trust me, I think it's an absolute travesty that we need jQuery but dropdown.min.js doesn't work without it.